litmus-portal/graphql-server/graph/image_registry.resolvers.go

```diff
@@ -22,7 +22,7 @@ func (r *mutationResolver) CreateImageRegistry(ctx context.Context, projectID st
 
 	ciResponse, err := imageRegistryOps.CreateImageRegistry(ctx, projectID, imageRegistryInfo)
 	if err != nil {
-		logrus.Error(err)
+		log.Error(err)
```

Comment:
Can you convert all
```go
if err != nil 
```
condition logs to `log.Fatal`?

Reply:
Hey @S-ayanide,
`log.Fatal` called `os.Exit(1)` so that process will exit with status set to 1. Is it okay?

https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

Reply:
Yes that is the expectation, if something in the middle goes wrong we'd want to terminate that function with a Fatal message. 

@amityt correct me if I'm wrong.

Reply:
No `log.Fatal` is not required here. It will impact the complete server. Rather a return statement can be added here in this error block.

Reply:
If we use `log.Fatal()` to exit the process, k8s Deployment obj restarts the pod automatically. IMO, It's an unnecessary task. Moreover, all of the logs are deleted so we cannot analyze what is the problem.